### PR TITLE
Image cropping

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR implement image cropping for TextBlocks.

Example with base-functionality:
------------------
screen1

Example with overlay-configuration:
--------------
screen1

Example with ratio-configuration
-------------
screen1

Example for cropping with limit configuration  (#481 )
---------------
screen1

Issuer: https://extranet.4teamwork.ch/support/basel-landschaft/tracker/52

### CODE REVIEW

File: ftw/simplelayout/images/cropping.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+class IImageCroppingMarker(Interface):
+    """
+    """
+
+
+@provider(IFormFieldProvider)
+class IImageCropping(model.Schema):
+
+    model.fieldset('image_cropping',
```
[the remark this answers is not in the document]
Comment (elioschmutz) [reply ftw/simplelayout/images/cropping.py]: 👍 moved

File: ftw/simplelayout/js/src/cropper/Cropper.js [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+    }
+}
+
+export default class Cropper {
+    constructor(imageCropperWrapper) {
+        this.HARD_LIMIT_IDENTIFIER = 'hard'
+        this.SOFT_LIMIT_IDENTIFIER = 'soft'
+
+        this.wrapper = $(imageCropperWrapper);
+        this.config = this.wrapper.data('config')
```
[the remark this answers is not in the document]
Comment (elioschmutz) [reply ftw/simplelayout/js/src/cropper/Cropper.js]: 👍 updated

File: ftw/simplelayout/resources.zcml [hunk 3]
```diff
@@ -14,6 +14,8 @@
                     after="ftw.theming:resources/scss/iconset_font-awesome.scss" />
 
         <theme:scss file="browser/resources/theming.toolbar-icons.scss" />
+
+        <theme:scss file="browser/resources/vendor/cropper.min.css" />
```
[the remark this answers is not in the document]
Comment (elioschmutz) [reply ftw/simplelayout/resources.zcml]: I can't reporduce this issue... 🤔 

I use the `pyScss-1.3.5-py2.7-macosx-10.11-x86_64.egg` package... is this the problem in your env?

**Comment by elioschmutz** [comment]
@jone Ok, I updated the PR.

The assertRatio will be restored properly now.

File: ftw/simplelayout/resources.zcml (hunk 3 above)
[the remark this answers is not in the document]
Comment (jone) [reply ftw/simplelayout/resources.zcml]: As discussed: we use the unminified version, that works.

**Comment by jone** [comment]
@elioschmutz -> https://github.com/4teamwork/ftw.simplelayout/commit/4c0bcdc93b7d20829910a5158b0f3154991df2ec

**Comment by elioschmutz** [comment]
Todo:
- [x] Textblock integration
- [ ] Sliderblock integration

**Comment by elioschmutz** [comment]
@jone Danke für dein Review!

Ich habe die Übersetzungen noch angepasst.

**Pull request merged**